src/components/AvatarWithIndicator.js

```diff
@@ -0,0 +1,51 @@
+import React, {memo} from 'react';
+import {View, StyleSheet} from 'react-native';
+import PropTypes from 'prop-types';
+import Avatar from './Avatar';
+import styles from '../styles/styles';
+
+const propTypes = {
+    // Is user active?
+    isActive: PropTypes.bool,
+    source: PropTypes.string.isRequired,
+    indicatorPosition: PropTypes.oneOf(['top-left', 'top-right', 'bottom-right', 'bottom-left']),
```

Comment:
This is a nice addition, but we don't need the ability to move the indicator around and should just hardcode the indicator to the `top-right` for now.

Reply:
ok. I will just use one.

Reply:
Forgive me @parasharrajat I meant to say "bottom right" like there is in the mockup.